demo/public/monocle-client-angular.js

```diff
@@ -75,69 +78,112 @@
-	                        if (!cached.hasOwnProperty(options.props[i])) {
-	                            this._cache.remove(path);
-	                            found = false;
-	                            break;
-	                        }
-	                    }
-	                    if (found) return Promise.resolve(cached);
-	                }
-	                break;
-
-	            // Remove from cache when resource is being updated or removed
-	            case 'post':
-	            case 'put':
-	            case 'delete':
-	            case 'patch':
-	                this._cache.remove(path);
-	                break;
-	        }
+	        return _handle.call(this, method, path, options);
+	    };
+	});
 
-	        var fullPath = buildFullPath(this._base, path);
-	        var query = buildQuery(options);
+	var updateBatchTimeout = function() {
+	    if (null !== this._batchTimeout) {
+	        // Batch timeout already set up, nothing to do.
+	        return;
+	    }
 
-	        if (query) fullPath += '?' + query;
+	    this._batchTimeout = setTimeout(function() {
+	        processBatch.call(this);
+	        this._batchTimeout = null;
+	    }.bind(this));
+	};
 
-	        return this._http.request(method.toUpperCase(), fullPath, options)
-	        .then(cacheResource.bind(this, method));
-	    };
+	var processBatch = function() {
+	    var batched = this._batched;
+	    this._batched = [];
 
-	    var cacheResource = function(method, resource) {
-	        if ('get' === method && resource.$id && resource.$expires) {
-	            this._cache.put(resource.$id, resource, resource.$expires);
-	        }
-	        return resource;
-	    };
+	    if (1 === batched.length) {
+	        var fullPath = buildFullPath(this._base, batched[0].url);
+	        var query = buildQuery(batched[0].options);
 
-	    var buildFullPath = function(base, path) {
-	        return (base + path).replace(/\/{2,}/g, '/');
-	    };
+	        if (query) fullPath += '?' + query;
 
-	    function buildQuery(options) {
-	        var query = {};
+	        return this._http.request(batched[0].method.toUpperCase(), fullPath, batched[0].options)
+	        .then(batched[0].resolve)
+	        .catch(batched[0].reject);
+	    }
 
-	        if (options && Array.isArray(options.props)) query.props = options.props.join(',');
+	    return this._http.request('POST', buildFullPath(this._base, '/_batch'), {
+	        body: batched
+	    }).then(function(results) {
+	        results.forEach(function(result, i) {
+	            if (result.status >= 200 && result.status < 300) {
+	                batched[i].resolve(result.body);
+	            } else {
+	                batched[i].reject(result.body);
+	            }
+	        })
+	    }).catch(function(error) {
+	        batched.forEach(function(apiCall, i) {
+	            batched[i].reject(error);
+	        });
+	    });
+	};
 
-	        var queryStringParts = [];
+	var _handle = function(method, path, options) {
+	    switch (method) {
```

Comment:
The `method` variable comes from a hard-coded array:

    ['get', 'post', 'put', 'patch', 'delete', 'options'].forEach(function(method) {